manifests/rbac/edit.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: flux-edit
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-edit: "true"
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
```

Comment:
Is this still necessary? [Back in August 2018 I learned](https://kubernetes.slack.com/archives/C0EN96KUY/p1533841633000133) from @deads2k that "view" aggregation is supposed to be included by "edit," and "edit" is supposed to be include by "admin" automatically.

Reply:
cert-manager and other controllers have these set. I guess it doesn't hurt to make this explicit so people reading the Flux RBAC understand what's going on.